sklearn/calibration.py

```diff
@@ -145,6 +146,13 @@ class CalibratedClassifierCV(ClassifierMixin, MetaEstimatorMixin, BaseEstimator)
 
         .. versionadded:: 0.24
 
+    class_weight : dict or 'balanced', default=None
+        Classes weights used for the calibration.
+        If a dict, it must be provided in this form: ``{class_label: weight}``.
+
+        Those weights won't be used for the underlying estimator training.
+        See :term:`Glossary <class_weight>` for more details.
+
```

Comment:
Thanks @amascia!
Could you please add a `versionadded` directive for the new parameter? Thanks!

Reply:
```suggestion

        .. versionadded:: 1.1
```